basic/atm/atm.py

```diff
@@ -0,0 +1,96 @@
+
+def delete_old_file(pfile):
+    """delete the old file of ATM data
+    """
+    pfile.seek(0)
+    pfile.truncate()
+    pfile.seek(0) # I believe this seek is redundant
+    return pfile
+def file_to_list(file):
+    """convert file to list
+        """
+    text = file.read()
+    list_1 = text.split('\n')
+    i = 0
+    list_2 = []
+    while i < len(list_1):
+        list_2 = list_2 + list_1[i].split(',')
+        i = i + 1
+    return list_2
+def dictionary_id_password(list):
+    """create dictionary id:password from list
+                """
+    i = 0
+    password_dct = {list[i]: list[i + 1] for i in range(0, len(list) - 1, 3)}
+    return password_dct
+def dictionary_id_money(list):
+    """create dictionary id:money from list
+            """
+    i = 0
+    money_dct = {list[i]: list[i + 2] for i in range(0, len(list) - 1, 3)}
+    return money_dct
+def get_password(id,password):
+    """check and change the password
+            """
+    inncorrect_flag = 1
+    while (inncorrect_flag == 1):
+        if input('input your password:\n') == password.get(id):
+            inncorrect_flag = 0
+            if input('Do you want your password?:\n') == 'yes':
+                password[id] = input('new password?:\n')
+        else:
+            print('incorrect password')
+            inncorrect_flag = 1
+
+    return  password
+def Cash_withdrawal(money):
+    """check cash withdrawal
+               """
+    if input('Do you want to do Cash withdrawal?:\n') == 'yes':
+        money[id] = float(money[id]) - float(input('how much Cash withdrawal:\n'))
```

Comment:
No reason to handle input and logic in the same line.

Reply:
I dont understand the comment

Reply:
Get user input in another line. In this line take care of the math and the math only.

Reply:
Is it critical or just for visibility?